src/evict/evict_lru.c

```diff
@@ -636,6 +636,9 @@ __evict_update_work(WT_SESSION_IMPL *session)
     } else
         LF_SET(WT_CACHE_EVICT_NOKEEP);
 
+    if (FLD_ISSET(conn->debug_flags, WT_CONN_DEBUG_UPDATE_RESTORE_EVICT)) {
+        LF_SET(WT_CACHE_EVICT_SCRUB);
```

Comment:
Is this doing all evictions as update restore? Or just the ones that are invoked by a `CURSOR::reset`?

Reply:
The plan is when the `WT_CONN_DEBUG_UPDATE_RESTORE_EVICT` flag is set, **all** evictions will be done as update restore

Reply:
If that's intentional, then fair enough. That's not what the ticket says though.

Reply:
Oops! have updated the ticket now